game-core/gradle.properties

```diff
@@ -1,2 +1,2 @@
-checkstyleMainMaxWarnings=446
+checkstyleMainMaxWarnings=431
```

Comment:
I just calculated, when all MemberName violations are fixed we are down to 233 checkstyle issues.